torch/linalg/__init__.py

```diff
@@ -367,6 +367,91 @@
             [-3.1113,  2.7381]], dtype=torch.float64)
 """)
 
+lstsq = _add_docstr(_linalg.linalg_lstsq, r"""
+torch.linalg.lstsq(input, b, cond=None, *, driver=None)
+    -> (Tensor solution, Tensor residuals, Tensor rank, Tensor singular_values)
+
+Computes the least squares solution to the system with a batch of matrices :math:`a` (represented by :attr:`input`)
```

Comment:
In a future PR we may want to make it clearer that this can accept both single matrices or batches.